doc/cmp.txt

```diff
@@ -342,17 +353,16 @@ Highlight                                                        *cmp-highlight*
 
 	Default: |CmpCompletionWindowBorder|
 
-*CmpDocumentationWindowScrollBar*
-  The highlight group for the background of the scrollbar; present when there
-  is no border on the documentation window.
+*CmpDocumentationWindowBordered*
+  The highlight group for a bordered documentation window.
 
-	Default: |CmpCompletionWindowScrollBar|
+	Default: |CmpDocumentationWindow|
 
-*CmpDocumentationWindowScrollThumb*
-  The highlight group for the thumb of the scrollbar on the documentation
-  window.
+*CmpDocumentationWindowPadding*
```

Comment:
What does this highlight group do? I don't really understand it, and it doesn't do anything no matter what colors I set it to, and also no matter if I have borders set or not

Reply:
Padding is for borders made of invisible/whitspace characters. It's used by default on the documentation window when no config is provided; try the following to see it in action:

```lua
window = {
  documentation = {
    winhighlight = {
      default = 'FloatBorder:Error',
    }
  }
}
```

There will be two columns of `Error` on either side of the documentation window after that, because we use spaces to provide padding.

Reply:
Hmm, okay I got it working. To me it feels like a really niche highlight group and kind of unnecessary, but maybe someone would want it🤷‍♂️

Reply:
We have to use use a highlight group there either way, it might as well be customizable like the others.